### PR TITLE
Follow up WebSocketTest OnError fix for Faces4.1

### LINKED ISSUE #28844: Follow Up On WebSocketTests in myfaces40_fat On Faces4.1 Release
Resolved #27598 for myfaces40, but we are still skipping the WebSocketTests on the EE11 repeat.  We will need to stop skipping this once faces4.1 is done.

### PR DESCRIPTION
**Pull request by tssmith04**
Fixes #28844 
The fix was added for Faces 4.1 as seen here: https://github.com/apache/myfaces/blob/47daecca509b07c580f9b139c11fc7fab1b3630f/api/src/client/typescript/faces/impl/PushImpl.ts#L156C9-L156C10

### CODE REVIEW

**Comment by tssmith04** [comment]
#build

**Comment by pnicolucci** [comment]
@tssmith04 @volosied  in my opinion the Faces 4.0 code should be moved to a new PR and associated with the original issue: https://github.com/OpenLiberty/open-liberty/issues/27598 this issue should also be a `release_bug` issue since this is a user visible bug that we are fixing. Then this PR should just have the new EE11 impl/FAT work in it.

**Comment by tssmith04** [comment]
#build

File: dev/io.openliberty.jakarta.faces.4.1/bnd.bnd [hunk 1]
```diff
@@ -31,7 +31,8 @@ DynamicImport-Package: \
 
 # Use this format (4.1.0.RC#)  once we have an official release and not an RC Version
 -includeresource: \
-	@${repo;org.apache.myfaces.core:myfaces-api;4.1.0.RC2;EXACT}!/!(META-INF/maven/*|module-info.class)
+	@${repo;org.apache.myfaces.core:myfaces-api;4.1.0.RC2;EXACT}!/!(META-INF/maven/*|module-info.class), \
+  META-INF=@src/META-INF
```
Comment (volosied) [inline dev/io.openliberty.jakarta.faces.4.1/bnd.bnd]: Spacing

**Review by volosied — DISMISSED** [review]
LGTM. thanks

**Review by volosied — APPROVED** [review]

**Pull request merged**